.github/workflows/build.yml

```diff
@@ -0,0 +1,113 @@
+name: Scala tests with coverage
+
+on:
+  pull_request:
+    paths-ignore: ['**.md']
+  push:
+    paths-ignore: ['**.md']
+
+jobs:
+  test:
+
+    runs-on: ubuntu-20.04
+
+    env:
+      SBT_OPTS: >-
+        -Xmx3g
+        -Denv.type=test
+        -Ddirectory.url=ldap://localhost:3389 -Ddirectory.password=testtesttest
+        -Dpostgres.host=localhost -Dpostgres.port=5432
+      ARTIFACTORY_USERNAME: ${{secrets.ARTIFACTORY_USERNAME}}
+      ARTIFACTORY_PASSWORD: ${{secrets.ARTIFACTORY_PASSWORD}}
+
+    services:
+      postgres:
+        image: postgres:9.6
+        env:
+          POSTGRES_USER: sam-test
+          POSTGRES_PASSWORD: sam-test
+          POSTGRES_DB: testdb
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 5432:5432
+
+    steps:
+      - uses: actions/checkout@v2
+
+      # coursier cache action caches both coursier and sbt caches
+      - name: coursier-cache-action
+        uses: coursier/cache-action@v5
+
+      - name: Set up opendj
+        run: |
+          sh docker/run-opendj.sh start
```

Comment:
do we know why travis had this:

```
- docker restart opendj
- sleep 40
```

and whether we need to do something similar (whether restarting or waiting 40 seconds for opendj to start up)? I see that it passed in this test but just checking in case it could cause a race condition and test flakiness.